build/stf-run-ci/tasks/create_catalog.yml

```diff
@@ -13,35 +13,76 @@
     sto_bundle_info: "{{ generate_bundle_sto.stdout_lines[-1] | from_json }}"
     sgo_bundle_info: "{{ generate_bundle_sgo.stdout_lines[-1] | from_json }}"
 
+- name: Generate default package names if not present
```

Comment:
Add this step inside the "Create info variables from provided pre-built bundles (deploy from bundles)" group of steps

Reply:
I think it's the opposite problem: the "deploy from bundles" group gets the package_name from the CSV, so this shouldn't run in that case at all. Changing the `when:` to `__local_build_enabled | bool and not __deploy_from_bundles_enabled | bool` so it only runs for local builds